samples/csharp_dotnetcore/13.core-bot/Bots/DialogBot.cs

```diff
@@ -18,10 +18,12 @@ namespace Microsoft.BotBuilderSamples.Bots
     public class DialogBot<T> : ActivityHandler
         where T : Dialog
     {
+#pragma warning disable CA1051 // Do not declare visible instance fields. We exclude this rule because the child class DialogAndWelcomeBot sohuld call its parant's RunAsync method.
         protected readonly Dialog Dialog;
         protected readonly BotState ConversationState;
         protected readonly BotState UserState;
         protected readonly ILogger Logger;
+#pragma warning restore CA1051 // Do not declare visible instance fields
```

Comment:
Do we really need this? can they just be priivate?, otherwise there iis a typo iin `sohuld ... parant's`